src/tools/adapter/CytoscapeAdapter.ts

```diff
@@ -0,0 +1,115 @@
+import { UbiquitousLanguageDto } from '../../model/DTO/UbiquitousLanguageDto';
+import { CytoscapeDto } from '../../model/DTO/CytoscapeDto';
+import { CytoscapeEdgeDto } from '../../model/DTO/CytoscapeEdgeDto';
+import { PatternFormatter } from './PatternFormatter';
+import { CytoscapePattern } from './CytoscapePattern';
+import { IAdapter } from './Adapter';
+import { UbiquitousLanguageJson } from '../../model/UbiquitousLanguageJson';
+import { NOT_ALLOWED_DEPENDENCIES } from './ErrorCodes';
+import { CytoscapeNode } from '../../model/CytoscapeNode';
+
+export class CytoscapeAdapter implements IAdapter {
+  private edgeCounter = 0;
+  private nodes: Array<CytoscapeNode>;
+  private edges: Array<CytoscapeEdgeDto>;
+  ubiquitousLanguageDto: UbiquitousLanguageDto;
+
+  constructor(ubiquitousLanguageJson: UbiquitousLanguageJson) {
+    this.ubiquitousLanguageDto = new UbiquitousLanguageDto(ubiquitousLanguageJson);
+    this.nodes = new Array<CytoscapeNode>();
+    this.edges = new Array<CytoscapeEdgeDto>();
+  }
+
+  public adapt() {
```

Comment:
the ubiquitousLanguageJson could be set as parameter of this method instead of the constructor.

Reply:
Yes, that's seems more legit